### PR TITLE
Prompt UI (command)

### PR DESCRIPTION
**Pull request by gabriel**
Takes update and options and calls prompt UI. The tests use mocked out responses.

### CODE REVIEW

**Comment by gabriel** [comment]
@keybase/updater-hackers

**Comment by gabriel** [comment]
Ignore the test failures.. it needs the config PR.

**Comment by chrisnojima** [comment]
@jzila

File: keybase/prompt_test.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+	}
+
+	options := cfg.updaterOptions()
+
+	promptOptions := updater.UpdatePromptOptions{AutoUpdate: false}
+
+	return ctx.updatePrompt(promptCommand, update, options, promptOptions)
+}
+
+func TestPromptApply(t *testing.T) {
```
Comment (chrisnojima) [inline keybase/prompt_test.go]: we need the timeout test

File: keybase/prompt_test.go [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+func TestPromptNoOutput(t *testing.T) {
+	resp, err := testPromptWithCommand(t, "echo")
+	assert.NoError(t, err)
+	if assert.NotNil(t, resp) {
+		assert.False(t, resp.AutoUpdate)
+		assert.Equal(t, updater.UpdateActionCancel, resp.Action)
+	}
+}
+
+func TestPromptError(t *testing.T) {
```
Comment (chrisnojima) [inline keybase/prompt_test.go]: can we have a test to return invalid json

File: keybase/prompt_test.go (hunk 1 above)
Comment (gabriel) [inline keybase/prompt_test.go]: Added test

File: keybase/prompt_test.go (hunk 2 above)
Comment (gabriel) [inline keybase/prompt_test.go]: Added test

**Comment by maxtaco** [comment]
I'll take a loko

**Comment by maxtaco** [comment]
👍

**Pull request merged**